### ISSUE TITLE
Gas.zip x Filecoin Grant

### ISSUE THREAD

**Issue by nic-maf** [filecoin-project/devgrants#1834]
# Open Grant Proposal: Gas.zip x Filecoin Integration
**Project Name:** Gas.zip
 
**Proposal Category:** Integrations
 
**Individual or Entity Name:** Size Research, Inc - Product: Gas.zip
 
**Proposer:** Size Research, Inc - Nicole Maffeo
 
**Project Repo(s)** - [Github - Gasdotzip](https://github.com/gasdotzip)
 
**(Optional) Filecoin ecosystem affiliations:** N/A

**(Optional) Technical Sponsor:** N/A
 
**Do you agree to open source all work you do on behalf of this RFP under the MIT/Apache-2 dual-license?:** Yes

# Project Summary:
We initially built Gas.zip to solve a pain point that we kept running into ourselves. Our team needed gas money for multiple upcoming alt chains and L2s. We were frustrated by existing infrastructure which is fragmented, inconsistent, and difficult to use. Bridges don’t support many chains, have different delivery times, require high minimums, and generally result in a ‘wandering in the forest’ experience.
 
We built Gas.zip to solve this problem. We decided to build a multi-chain distributor that could support as many chains as possible while delivering a familiar, friendly and simple user experience. We also wanted near instant delivery times to reduce user overhead when swapping around on dozens of different chains.
 
Today, Gas.zip is live and in production, operating for 18 months with thousands of daily active users. Gas.zip is the fastest one-stop instant liquidity bridge and refuel station for over 300+ chains. Gas enables cheap and near instant bridging for users looking to navigate Ethereum L2's, alt-chains and altVMs. Users can bridge to multiple destination blockchains with a single inbound transaction leveraging the multi send functionality. Gas is designed and optimized to reduce gas costs and delivery times traditionally associated with bridging.
 
# Impact:
As mentioned above, crypto infrastructure is fragmented and has limited UI. It's difficult for new users and even crypto native users to find and bridge between new chains, especially across VMs. Gas.zip makes chain exploration easy. Users are given hundreds of options of ecosystems to discover and a means to travel to them both quickly and cheaply.
 
# Outcomes:
There are a lot of metrics to measure success here. One really interesting success metric we like to look at is the number of chains people are bridging to a new ecosystem from, especially through routes that didn’t previously exist. Other more routine metrics probably include: total number of users/addresses, user growth month over month, average dollar amount bridged by user, increase in average amount bridged through time. It's also always helpful to count any specific integrations with say Filecoin specific dapps.
 
# Development Roadmap:
#### **Milestone 1:** Add Filecoin mainnet as an inbound x outbound chain | 1/1/2025 | $20,000
 
Filecoin mainnet will be added as an inbound and destination chain for users, with a marketing release along with upkeep (RPCs, direct user support, etc.)
 
#### **Milestone 2:** Add Filecoin testnet and devnet as outbound chains | 2/1/2025 | $10,000
 
Filecoin testnet and devnet will be added as destination chain for users, with a marketing release along with upkeep (RPCs, direct user support, etc.)
 
#### **Milestone 3:** Run 10 Filecoin-specific marketing campaigns | 6/1/2025 | $10,000
 
Highlight relevant metrics as well as do deep dives on any Filecoin-specific dApps and chain-specific properties.
 
#### **Milestone 4**: Hit specific growth metrics | 6/1/2025 | $10,000
 
* 25,000 unique transactions to Filecoin Mainnet
* $50,000 total volume bridged to Filecoin Mainnet
* 500 unique addy interactions with devnet /tesnet
 
# Total Budget Requested:
* $50,000
 
# Maintenance and Upgrade Plans:
We've been building Gas.zip for 18+ months. Gas.zip user growth has been on a steep upward trajectory and we are excited to continue shipping new features and integrations for the foreseeable. In addition to shipping new features and supporting new chains, we look forward to deepening relationships with existing ecosystem partners. We have partnerships with a number of foundations and are working closely to onboard new users to their ecosystems, integrate with new dapps and generally make d2d bridging easier for existing users.
 
# Team Members:
- [Nicole Maffeo](x.com/nicole_maffeo) - Founder
- [Tom Kysar](x.com/tomkysar) - Founder
- [Ragepit](x.com/ragepit) - Backend
- [Seranged](x.com/seranged) - Frontend
- [Bruno Confortin](x.com/bconfortin) - Middeware
- [Blockhead](x.com/ohyoublockhead) - Marketing

# Team Member LinkedIn Profiles:
- [Nicole Maffeo](https://www.linkedin.com/in/nicole-maffeo/) - Founder
- [Tom Kysar](https://www.linkedin.com/in/kysar/) - Founder
 
# Team Website:
- [Gas.zip](https://www.gas.zip/)
 
# Relevant Experience:
**Tom Kysar** has been working with crypto startups since 2014, A decentralization maxi, he has handled product, engineering, operations and more for various crypto startups for over a decade, such as Koinify, Tierion and Augur. 
 
**Nicole Maffeo** began her career on Ray Dalio's executive team at Bridgewater Associates. She later co-founded an AI computer vision startup acquired by Google Research, where she managed a large AI operations and infrastructure team, launching over a dozen products in NLP, CV, and multi-modal domains. Nicole started exploring crypto in 2019 and co-founded SIZE in 2022.
 
**Seranged** is a frontend engineer who in addition to development, handles developer relations, business development, developer evanglaism an more. 
 
**Ragepit** is a backend enegineer and Ethereum native developer, specalizing in traditional backend infrastructure along with everything on-chain. As a frequent participant in ecosystem CTFs, you may see his name on the top of a leaderboard. 
 
**Bruno** is a frontend and middelware engineer, with a strong web2 and crypto background, working prior at projects such as Augur and Dapper Labs. 
 
**Blockhead** is head of marketing and strategy. As a longtime crypto native, you have most likey seen his tweets on your timeline, with relevant experience working for a handful of highly popular crypto applications prior to Gas. 

# Team Code Repositories:
 
- [Gas.zip Contracts](https://github.com/gasdotzip/gas-contracts)
- [Gas.zip LZ Contracts](https://github.com/gasdotzip/gas-lz-contracts)
 
# Additional Information:
We learned about the program from Aysun Demircan who worked at Filecoin / Protocol Labs. Fomer colleague with Nicole Maffeo (Worked closely).

**Comment by nic-maf** [comment]
Yes of course. Looks like the twitter links didn't stick. See below. Thanks so much. 

[Nicole Maffeo](https://x.com/nicole_maffeo) - Founder
[Tom Kysar](https://x.com/tomkysar) - Founder
[Ragepit](https://x.com/rage_pit) - Engineering
[Seranged ](https://x.com/Seranged)- Engineering
[Bruno Confortin ](https://x.com/bconfortin)- Engineering
[Blockhead](https://x.com/OhYouBlockhead) - Marketing

**Comment by ErinOCon** [comment]
Hi @nic-maf, can you share more on how this project provides value over buying tokens on an exchange? Also, have you completed security audits for this project work? 

Many thanks for your time with these details!

**Comment by ErinOCon** [comment]
Hi @nic-maf, flagging that our team will be on holiday starting December 21st. Do you anticipate having an update available before this date?

**Comment by ErinOCon** [comment]
Hi @nic-maf! Should we expect a new update from your team?

**Comment by ErinOCon** [comment]
@nic-maf This Issue has been closed. If you would like to proceed with our grant review process, please feel welcome to submit a new proposal [here](https://github.com/filecoin-project/devgrants/issues/new/choose).
